FitNesseRoot/DbFit/AcceptanceTests/CoreTests/StoredProcNoParams/content.txt

```diff
@@ -1,5 +1,9 @@
 !3 If there are no parameters, Execute Procedure needs just one row
 
+|Clean|
+|table|clean?|
+|users|true|
```

Comment:
I wonder if we can add a `TearDown` page to `CoreTests` which to do the necessary cleanup. (And include in case of Informix; and possibly other similar cases in future).